sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/GatewayConnectionConfig.java

```diff
@@ -130,7 +135,8 @@ public String toString() {
             "requestTimeout=" + requestTimeout +
             ", maxConnectionPoolSize=" + maxConnectionPoolSize +
             ", idleConnectionTimeout=" + idleConnectionTimeout +
-            ", inetSocketProxyAddress=" + inetSocketProxyAddress +
+            ", proxyType=" + proxy.getType() +
```

Comment:
should I also add the proxy type?

Reply:
to the `toString()` - yes.

Reply:
just found out that the spotbug checks all the properties should be referenced in the toString(), so added the following exclusion:
  <Match>
    <Class name="com.azure.cosmos.com.azure.cosmos.GatewayConnectionConfig"/>
    <Field name="proxy"/>
    <Bug pattern="UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR"/>
  </Match>

Reply:
I am wondering why this wasn't a problem with `InetSocketAddress` ?
It was also present in `toString()` and was not initialized in Constructor.

Reply:
oo, I see, I think the UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR was really checking is that I used proxy.getType, proxy.getAddress, but it might never be inistialized, so may get null pointer exception. 

Updated to check proxy is null or not.